CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ cmake_policy(SET CMP0077 NEW)
 set(CMAKE_MACOSX_RPATH 1)
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 set(CMAKE_OSX_DEPLOYMENT_TARGET 11)
+set(CMAKE_FIND_PACKAGE_PREFER_CONFIG 1)
```

Comment:
This should be tied to `USE_VCPKG` though. Otherwise it will be broken. Why not just use the `USE_VCPKG` flag in the fine.cmake files? Maybe it needs to be made global or something. I've never really tried it.

Reply:
> Otherwise it will be broken

Hmm, what do you mean by that?

Reply:
Becaues if you specify `USE_VCPKG` as true and this flag as false, then it won't work correctly no? Whatever `USE_VCPKG` is set as should be what this flag also is set to.

Reply:
Hm, still don't understand. As I understand it, vcpkg tends to provide "config" modules, while csound provides "find" modules. So we have three options:

1) Only load the csound "find" modules if vcpkg isn't available
2) Specify whether we want a "config" or a "find" module depending on whether or not we are using vcpkg
3) Choose to always use a config module if it's available

Reply:
USE_VCPKG should be the main flag to decide if the build is using it or not. If you introduce another flag, then it's possible to have USE_VCPKG=1 and CMAKE_FIND_PACKAGE_PREFER_CONFIG as 0, which wouldn't work. Just use the USE_VCPKG flag

Reply:
Hmm, no sorry I still don't understand. `CMAKE_FIND_PACKAGE_PREFER_CONFIG` is an internal cmake setting, not a new option I'm adding. It just tells cmake that if there is both a `find` and a `config` package available, use the `config` one.

Reply:
Ah ok. I wasn't paying attention to that. Then this make sense. I wasn't aware of that option. Fair play